page/partials/commit_funds.ejs

```diff
@@ -1,4 +1,4 @@
-<div class="commit-funds section">
+<div class="commit-funds section" id="commit">
```

Comment:
This is done so that we can redirect users to this section when they press on the footnotes